ledger/src/blockstore_processor.rs

```diff
@@ -945,17 +937,24 @@ fn load_frozen_forks(
                 cache_block_time_sender,
                 None,
                 timing,
-            )
-            .is_err()
-            {
+            );
+            // Insert even dead banks into the BankForks so that cleanup
+            // of account state created on Bank creation via `Bank::new_from_parent()`
+            // will occur when the bank is dropped. Otherwise, this state is not
```

Comment:
So, `purge_slot()` called inside `Bank::drop()` isn't enough for `cleanup of account state`?

Reply:
oh, maybe this?: https://github.com/solana-labs/solana/pull/17269

Reply:
@ryoqun, hehe good eye. Yeah, it used to only purge the storage entries, not the accounts index. Maybe with this change here: https://github.com/solana-labs/solana/pull/17319, this change in blockstoreprocessor can now be avoided

Reply:
@ryoqun with https://github.com/solana-labs/solana/pull/17319, I think this change is no longer necessary, so reverted the `blockstore_processor` changes!